packages/react-app-revamp/pages/contest/[chain]/[address]/rules/index.tsx

```diff
@@ -76,13 +78,20 @@ const Page: NextPage = (props: PageProps) => {
        </ul>
      </section>
      <section>
-      <h2 className='uppercase font-bold mb-2'>Token</h2>
+      <h2 className='uppercase font-bold mb-2'>Voting token</h2>
```

Comment:
Can we put this below the submission token info just to keep the flow of submission -> voting?